src/tablecellproperties/tablecellpropertiesui.js

```diff
@@ -121,7 +121,11 @@ export default class TableCellPropertiesUI extends Plugin {
 		} );
 
 		this.listenTo( view, 'cancel', () => {
-			editor.execute( 'undo', this._undoStepBatch );
+			// https://github.com/ckeditor/ckeditor5/issues/6180
+			if ( this._undoStepBatch.operations.length ) {
```

Comment:
I'm not sure I like this check but for now it is OK as it uses pulbic batch API. https://github.com/ckeditor/ckeditor5/issues/6212